test/libsolidity/syntaxTests/parsing/enum_with_256_members.sol

```diff
@@ -0,0 +1,29 @@
+enum E {
```

Comment:
Can you make this a semantic test and check that the values for E00 and E255 work as expected?

Reply:
And that the abi decoder v2 rejects 256 as a value?

Reply:
Done.